run.py

```diff
@@ -0,0 +1,11 @@
+#!flask/bin/python
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return "Hello, World!"
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
Errrm.... Is there a reason why this file is here? I believe it's a mistake right?